fvm/src/call_manager/default.rs

```diff
@@ -315,12 +328,26 @@ where
                     .map_err(Abort::Fatal)?;
 
                 // Invoke it.
-                let return_block_id = invoke.call(&mut store, (param_id,))?;
+                let res = invoke.call(&mut store, (param_id,));
+
+                // First, charge gas for the "latest" use of execution units (all the exec units used since the most recent syscall)
```

Comment:
We should clarify that the variable below is "total units consumed".

Reply:
Ack.